usecases/MOC_integration/moc_integration.adoc

```diff
@@ -2,80 +2,82 @@
 
 == Abstract
 
+The content of this repo is under Openshift 3.x, now it is under the process to be updated to version 4.x.
+
 This document describes the steps that are required to get ChRIS with `pfcon` and `pman` up and running on the MOC. The intended audience is ChRIS developers.
 
 . xref:#registering-to-mass-open-cloud[Registering to Mass Open Cloud]
-. xref:#adding-projects-to-openshift-and-openstack[Adding Projects to Openshift and Openstack]
+. xref:#adding-projects-to-openshift[Adding Projects to Openshift]
 . xref:#installing-openshift-cli-tool[Installing Openshift CLI Tool]
 . xref:#creating-secrets-on-openshift-using-cli-tool[Creating Secrets on Openshift Using CLI Tool]
 . xref:#deploying-pman-on-openshift[Deploying pman on Openshift]
 . xref:#deploying-pfcon-on-openshift[Deploying pfcon on Openshift]
-. xref:#running-test-scripts-on-openshift[Running Test Scripts on Openshift]
 . xref:#additional-step-building-a-local-openshift-cluster-via-crc[(Additional Step) Building a Local Openshift Cluster via CRC]
-. xref:#alternative-way-building-a-local-openshift-cluster-via-openshift-origin[(Alternative Way) Building a Local Openshift Cluster via OpenShift Origin]
+
 
 == Registering to Mass Open Cloud
 
 * In order to create an account, you should submit a request to MOC. It usually takes 2-3 business days to get approved. + 
-Follow this link -> https://massopen.cloud/request-an-account/[Request Account on MOC]
+Follow this link -> https://massopen.cloud/request-an-account/[Request Account on MOC]. The information provided in the previous link is ambiguous. Note that only OpenStack has been moved to NERC, OpenShift function is still on MOC and the registration should be working.
 
-* Learn more about adding users to your Openstack project. Follow this link -> https://support.massopen.cloud/kb/faq.php?id=22[Add Users to Openstack Project]
+* Learn more about adding users to your OpenShift project. Follow this link -> https://docs.openshift.com/container-platform/3.11/admin_guide/manage_users.html[Add Users to OpenShift Projects]
```

Comment:
This is a link to OpenShift 3.11 documentation which is not up to date with OpenShift 4.

Adding users to OpenShift projects on MOC is handled via helpdesk. https://osticket.massopen.cloud/